apps/dashboard/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,57 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { EthereumWalletGuard, NoEthereumWalletGuard } from '@nft/metamask';
+
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: 'landing',
+    pathMatch: 'full'
+  },
+  {
+    path: 'landing',
+    loadChildren: () => import('./landing/landing.module').then(m => m.LandingModule)
+  },
+  {
+    path: 'account',
+    canActivate: [EthereumWalletGuard],
+    loadChildren: () => import('../../../marketplace/src/app/my-tokens/my-tokens.module').then(m => m.MyTokensModule)
+  },
+  {
+    path: 'contact',
+    loadChildren: () => import('../../../marketplace/src/app/contact-us/contact-us.module').then(m => m.ContactUsModule)
+  },
+  {
+    path: 'marketplace',
+    loadChildren: () => import('../../../marketplace/src/app/marketplace/marketplace.module').then(m => m.MarketplaceModule)
+  },
+  {
+    path: 'faq',
+    loadChildren: () => import('../../../marketplace/src/app/faq/faq.module').then(m => m.FAQModule)
+  },
+  {
+    path: 'signin',
```

Comment:
we dont need a signin route, same for faq and marketpalce and contact

Reply:
same for 'account' as My-tokens?

Reply:
going to be a different component that @RemcoSimonides  is working on

Reply:
>we dont need a signin route
So guards have to be updated also